docs/kcl-lang/known-issues.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "KCL Known Issues"
+title: "Known Issues"
```

Comment:
FYI, when I've changed or added these in the past, I had to update the website repo since it looks for these to put them at the top of the sidebar.